src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -259,27 +260,30 @@ def suppress_one_group_saturated_ramps(ramp_data):
     dq = ramp_data.groupdq
     nints, ngroups, nrows, ncols = dq.shape
     sat_flag = ramp_data.flags_saturated
+    jump_flag = ramp_data.flags_jump_det
 
     ramp_data.one_groups = [None] * nints
 
     for integ in range(nints):
         ramp_data.one_groups[integ] = []
         intdq = dq[integ, :, :, :]
-
-        # Find ramps with a good zeroeth group, but saturated in
-        # the remainder of the ramp.
-        wh_one = groups_saturated_in_integration(intdq, sat_flag, ngroups - 1)
+        # Find ramps with only one group that is not saturated and
+        # not jump (ie only one good group)
```

Comment:
along with saturation and jump, could 'do not use' or 'ad floor' flags also cause this scenario?